.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+*Descriptive Summary of your Bugfix, Feature, or Refactoring.*
```

Comment:
I think we should write the nouns in this sentence in lowercase. There's again some German bleeding into English 😂